redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/requests/AllocateDatabaseServerV4Request.java

```diff
@@ -29,11 +29,7 @@
 
     @NotNull
     @ApiModelProperty(value = ModelDescriptions.ENVIRONMENT_ID, required = true)
```

Comment:
Now is a good time to replace ModelDescriptions.ENVIRONMENT_ID with ENVIRONMENT_CRN.

Reply:
Thanks, done.